### PR TITLE
Switch to async GA tag

### LINKED ISSUE #6253: P1 - Switch to async GA tag
From https://developers.google.com/analytics/devguides/collection/analyticsjs/#alternative_async_tag

*"The alternative async tag below adds support for preloading, which will provide a small performance boost on modern browsers, but can degrade to synchronous loading and execution on IE 9 and older mobile browsers that do not recognize the async script attribute. Only use this tag configuration if your visitors primarily use modern browsers to access your site."*

That's us!

So, instead of 
```
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-XXXXX-Y', 'auto');
ga('send', 'pageview');
</script>
```

we switch to 
```
<script>
window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
ga('create', 'UA-XXXXX-Y', 'auto');
ga('send', 'pageview');
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>
```
which means that when the browser sees that second script tag it can start to download the file before it starts executing the (second) script tag.

### PR DESCRIPTION
**Pull request by mindy**
Decided to load the async script tag for everyone. Conclusion reached in https://github.com/mdn/kuma/issues/6253#issuecomment-584340062

Closes #6253

### CODE REVIEW

File: jinja2/includes/google_analytics.html [hunk 1]
```diff
@@ -67,3 +63,4 @@
     }
     {%- endif %}
 </script>
+<script async src='https://www.google-analytics.com/{{ GA_FILE }}.js'></script>
```
Comment (schalkneethling) [inline jinja2/includes/google_analytics.html]: Nit: missing EOL

File: jinja2/includes/react_google_analytics.html [hunk 2]
```diff
@@ -7,12 +7,10 @@
...
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','https://www.google-analytics.com/{{ GA_FILE }}.js','ga');
+        window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
         ga('create', '{{ settings.GOOGLE_ANALYTICS_ACCOUNT }}', 'mozilla.org');
         ga('set', 'anonymizeIp', true);
     }
     {%- endif %}
 </script>
+<script async src='https://www.google-analytics.com/{{ GA_FILE }}.js'></script>
```
Comment (schalkneethling) [inline jinja2/includes/react_google_analytics.html]: Nit: missing EOL

**Comment by peterbe** [comment]
Awesome. So DNT is still respected. The worst that can happen is that Google can notice that someone downloaded the .js file.

**Comment by schalkneethling** [comment]
Tested with DNT enabled. Without DNT enabled something like the following is printed to the console:

```
Running command: ga("send", {hitType: "event", eventCategory: "Wiki", eventAction: "MainNav", eventLabel: "Technologies▼"})
analytics_debug.js:25 Storage not available. Aborting hit.
```

with DNT enabled, it prints something like the following:

```
Running command: ga("send", {hitType: "event", eventCategory: "Wiki", eventAction: "MainNav", eventLabel: "Technologies▼"})
analytics_debug.js:25 Command ignored. Unknown target: undefined
```

o/\o

With DNT and PrivacyBadger, the analytics script is completely blocked, but all functionality still works.

**Comment by schalkneethling** [comment]
@mindy just the two `EOL` items. Other than that, merge at will 🍵

File: jinja2/includes/google_analytics.html (hunk 1 above)
Comment (mindy) [reply jinja2/includes/google_analytics.html]: Fixed!

File: jinja2/includes/react_google_analytics.html (hunk 2 above)
Comment (mindy) [reply jinja2/includes/react_google_analytics.html]: Fixed! And will add a setting to automatically add a new line in vscode

**Comment by mindy** [comment]
Thank you for reviewing @schalkneethling !

**Comment by callahad** [comment]
@mindy Let's un-revert this, since the perf metrics haven't changed after deploying ca984b2f61858c939e6b508f253bd69c17609cc9

**Comment by mindy** [comment]
Got it-- opened https://github.com/mdn/kuma/pull/6621

**Pull request merged**